### PR TITLE
Add the `ChunkedEvaluations` type

### PR DESCRIPTION
**Pull request by duguorong009**
**What's been done:**
- Add a new type `ChunkedEvaluations` for the `ProofEvaluations` refactor

Reference: #759

### CODE REVIEW

**Comment by duguorong009** [comment]
@mimoo 
Can you please review the PR?
Thanks in advance!

File: utils/src/chunked_evaluations.rs [hunk 1]
```diff
@@ -0,0 +1,17 @@
+//! This module contains a type [ChunkedEvaluations],
+
+use serde::{Deserialize, Serialize};
+use serde_with::serde_as;
+
+/// This struct contains multiple chunk evaluations.
+#[serde_as]
+#[derive(Clone, Serialize, Deserialize)]
+#[serde(bound = "F: ark_serialize::CanonicalDeserialize + ark_serialize::CanonicalSerialize")]
```
Comment (mimoo) [inline utils/src/chunked_evaluations.rs:9]: I don't think you need this bound no?

Comment (duguorong009) [reply utils/src/chunked_evaluations.rs:9]: Um, I think that bound restricts the possible types of `F`.
My first attempt was like 
```
pub struct ChunkedEvaluations<F: ark_ff::Field> {
...
```
I think we can restrict the `F` type bound by using above `serde(bound)`

Comment (mimoo) [reply utils/src/chunked_evaluations.rs:9]: oh right, I think you should just add `pub struct ChunkedEvaluations<F> where F: PrimeField { ...` which is simpler : o

Comment (duguorong009) [reply utils/src/chunked_evaluations.rs]: Done https://github.com/o1-labs/proof-systems/pull/840/commits/1de06fb10d51ac2bfffddd1a622938ab704079b4

**Review by mimoo — APPROVED** [review]
nice! thanks a lot :)

**Pull request merged**